tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_included_record_parameters_negative.bal

```diff
@@ -194,3 +194,67 @@ function functionWithIncludedRecordParam14(int a, *Options options, int c = 10)
 function functionWithIncludedRecordParam15(int a, int b = 12, *Options options, int c = 10) {
     string? name = options?.name;
 }
+
+type Record1 record {
+    int id;
+    string name;
+};
+
+function fn1(*Record1 record1) {
+
+}
+
+type Record2 record {|
+    int id?;
+    string name?;
+|};
+
+function fn2(*Record2 record2) {
+
+}
+
+function functionWithIncludedRecordParam16() {
+    fn1(id = 2, record1 = {id: 2, name: ""}); // error
```

Comment:
Shouldn't we be using `{id: 2, name: ""}` 's pos for the error pos since value for field `id` is actually duplicated there?

Reply:
I think it is matched with the error message we use,
`duplicate arguments may have been specified for ''{0}'' via the named argument and the argument for the \
  included record parameter`